src/app/@theme/components/footer/footer.component.ts

```diff
@@ -4,13 +4,7 @@ import { Component } from '@angular/core';
   selector: 'ngx-footer',
   styleUrls: ['./footer.component.scss'],
   template: `
-    <span class="created-by">Created with ♥ by <b><a href="https://akveo.com" target="_blank">Akveo</a></b> 2017</span>
-    <div class="socials">
-      <a href="#" target="_blank" class="ion ion-social-github"></a>
-      <a href="#" target="_blank" class="ion ion-social-facebook"></a>
-      <a href="#" target="_blank" class="ion ion-social-twitter"></a>
-      <a href="#" target="_blank" class="ion ion-social-linkedin"></a>
-    </div>
+    <span class="created-by">Using template created by <b><a href="https://akveo.com" target="_blank">Akveo</a></b> 2017</span>
```

Comment:
to chyba też do wyrzucenia czy nie ?

Reply:
Właśnie nie wiem, jak jest licencjonowana templatka, czy można wywalić dane o autorach.